README.md

```diff
@@ -30,12 +30,43 @@ docker compose down
 docker compose -d --build
 ```
 
+### Running the applications locally (without docker)
+
+You may still need docker to start the databases.
+
+From the root directory,
+`docker compose up postgres`
+
+This will start the postgres container running on port 5432; with username `user` and password `root`,
+and a database created called `notarial` (`postgres://user:root@localhost:5432/notarial`).
+[`init.sql`](init.sql) is also loaded as a volume into the container, and will create a database named `queue`
+(`postgres://user:root@localhost:5432/notarial`). This is useful if you are going to be running [forms-worker](https://github.com/UKForeignOffice/forms-queue)
```

Comment:
This should say `postgres://user:root@localhost:5432/queue` right?

Reply:
No, technically both are correct but this intentionally this matches how our RDS instances is set up, this repo should only really care about /notarial. 

/queue is reserved for forms-queue/forms runner related activities, and /notarial is for any messages emitted from this repo.

In this repo's docker compose we have 

```
  postgres:
    container_name: postgres
    image: "postgres:16"
    volumes:
      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
    ports:
      - "5432:5432"
    environment:
      POSTGRES_DB: notarial
      POSTGRES_PASSWORD: root
      POSTGRES_USER: user
```

which creates a db at postgres://user:root@localhost:5432/notarial.

I've changed all of the environment variables (I think) to use postgres://user:root@postgres:5432/notarial or postgres://user:root@localhost:5432/notarial.


but, 

```
 volumes:
      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
```
init.sql also creates a database at /queue for convenience.

Reply:
oh sorry, I see where the comment is meant to be pointing at now